lib/adapters/MochaAdapter.js

```diff
@@ -1,16 +1,19 @@
 import EventEmitter from 'events'
-import { Test, Suite } from '../Data.js'
+import {Test, Suite} from '../Data.js'
 
 export default class MochaAdapter extends EventEmitter {
   constructor (mocha) {
     super()
 
     this.mocha = mocha
+
     mocha.reporter((runner) => {
       this.runner = runner
+
       runner.on('start', this.onStart.bind(this))
       runner.on('suite', this.onSuite.bind(this))
       runner.on('test', this.onTest.bind(this))
+      runner.on('pending', this.onPending.bind(this))
```

Comment:
I think it is covered, because there are 2 skipped tests in Mocha's test fixture. Through this change I only moved [some code](https://github.com/js-reporters/js-reporters/pull/66/commits/7c56c0aaa20dc84b2a8f45232b63c737fcb72526) from `onTestEnd` function into the new `onPending` function.